client/apis/api.ts

```diff
@@ -24,3 +24,15 @@ export async function getUser(token: string) {
 
   return res.body as Profile
 }
+
+export async function fetchCrewList(token: string) {
+  const response = await request
+    .get('/api/v1/crews/')
+    .set('Authorization', `Bearer ${token}`)
+  return response.body
+}
+
+export async function fetchEventList(crewId: number) {
```

Comment:
alter this to look like the function on line 28: 

const response = await request 
.get('api route')
.set('Authorization', `Bearer ${token}`
return response.body